.github/workflows/test-sdk.yml

```diff
@@ -27,24 +27,27 @@ jobs:
         with:
           node-version: '16.13.0'
 
-      - name: restore lerna
-        uses: actions/cache@master
+      - name: Cache pnpm modules
+        uses: actions/cache@v2
         with:
-          path: |
-            node_modules
-            **/node_modules
-            /home/runner/.cache/Cypress
-          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6.0.2
+          run_install: true
 
       # Runs a single command using the runners shell
       - name: Install project dependencies
-        run: yarn install
+        run: pnpm install
```

Comment:
Same here with the duplicate install

Reply:
Yes, I missed that one I will update shortly.